frontend/src/admin/common/AddButton.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
+import styled from 'styled-components';
+import IconButton from './IconButton';
 
-const AddButton = ({ onClick, title = 'Add', children }) => (
-  <button onClick={onClick} className="new" title={title}>
+const StyledAddButton = styled(IconButton)`
+  &::after {
+    content: "\\f067";
+    font-family: FontAwesome;
+  }
+`;
+
+const AddButton = ({ onClick, children }) => (
+  <StyledAddButton onClick={onClick} title="Add">
     {children}
-  </button>
+  </StyledAddButton>
 );
 
 export default AddButton;
```

Comment:
This is a great example of how styled-components naturally leads you to create separate smart vs. presentational components. 👍 

Also with this pattern of lots of little styled primitives, I can see now how Facebook ended up with thousands of components.